widget/ui/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,58 @@
+import type { PropTypes } from './Tabs.types';
+
+import React from 'react';
+
+import { Divider } from '../Divider';
+import { Tooltip } from '../Tooltip';
+
+import { BackdropTab, Tab, Tabs } from './Tabs.styles';
+
+export function TabsComponent(props: PropTypes) {
+  const {
+    Items,
+    onChange,
+    container = document.body,
+    value,
+    currentTabIndex,
+    tabWidth,
+    type,
+    border,
```

Comment:
I think border is not a good name for a prop which only manipulates the borderRadius of the container and items.

Reply:
fixed